Ghidra/Features/Base/src/main/java/ghidra/app/util/bin/format/omf/OmfRecord.java

```diff
@@ -247,6 +249,6 @@ public final static String getRecordName(int type) {
 	@Override
 	public String toString() {
 		return String.format("name: %s, type: 0x%x, offset: 0x%x, length: 0x%x",
-			getRecordName(recordType & 0xfe), recordType, recordOffset, recordLength);
+			getRecordName(recordType & ~1), recordType, recordOffset, recordLength);
```

Comment:
I think `(byte) 0xfe` would read a bit better.

Reply:
I will test if it works.